e2e/provision/install_sandbox.sh

```diff
@@ -89,9 +89,17 @@ if [ "${DEPLOYMENT_TYPE:-r1}" == "one-summit" ]; then
 else
     # Management cluster creation
     if [[ ${DEBUG:-false} != "true" ]]; then
-        ansible-playbook -i 127.0.0.1, playbooks/cluster.yml
+        if [ ${#DOCKERHUB_USERNAME} != 0 ] && [ ${#DOCKERHUB_TOKEN} != 0 ]; then
+            ansible-playbook -i 127.0.0.1, playbooks/cluster.yml -e "DOCKERHUB_USERNAME=$DOCKERHUB_USERNAME DOCKERHUB_TOKEN=$DOCKERHUB_TOKEN"
+        else
+            ansible-playbook -i 127.0.0.1, playbooks/cluster.yml
+        fi
```

Comment:
@vjayaramrh I think those changes are not required anymore

Reply:
I see, can you please elaborate.
I remember I had to add those changes as it was failing in the openstack VM in our lab which has multiple tenants and was hitting the docker rate limit. This PR is pushed after hitting the issue in the environment and testing out in the same environment that it fixes the issue.

Reply:
Nevermind, I saw that you didn't include my suggestions.